test/testkit/assert/logs.go

```diff
@@ -96,3 +96,12 @@ func LogPipelineConditionReasonsTransition(ctx context.Context, k8sClient client
 		fmt.Fprintf(GinkgoWriter, "Transitioned to [%s]%s\n", currCond.Status, currCond.Reason)
 	}
 }
+
+func LogPipelineUnsupportedMode(ctx context.Context, k8sClient client.Client, pipelineName string, unsupportedModeBool bool) {
```

Comment:
you don't need to suffix the type to the argument name in `unsupportedModeBool`
Maybe, we can rename it to `isUnsupportedMode`